apps/sms/views/conversation/js/conversation.js

```diff
@@ -513,16 +513,16 @@ var ConversationView = {
     // get an event whenever the panel changes?
     Threads.currentId = args.id;
 
-    var prevPanel = args.meta.prev && args.meta.prev.panel;
+    var prevPanel = args.meta.prev;
 
     // If transitioning from composer, we don't need to notify about type
     // conversion but only after the type of the thread is set
     // (afterEnterThread)
-    if (prevPanel !== 'composer') {
+    if (!prevPanel || prevPanel.panel !== 'composer') {
```

Comment:
I admit these "double" checks look ugly, but can't think of smth better and not risky